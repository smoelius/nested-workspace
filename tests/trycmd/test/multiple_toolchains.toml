args = ["test", "--workspace"]

[bin]
name = "cargo"

[fs]
cwd = "../../../fixtures/multiple_toolchains"
