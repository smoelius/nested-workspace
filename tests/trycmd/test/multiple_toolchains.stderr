...
   Compiling multiple_toolchains v0.1.0 ([CWD])
    Finished `test` profile [unoptimized + debuginfo] target(s) in [..]s
     Running unittests src/lib.rs (target/debug/deps/multiple_toolchains-[..])
<<< [CWD]/beta
...
   Compiling beta v0.1.0 ([CWD]/beta)
    Finished `test` profile [unoptimized + debuginfo] target(s) in [..]s
     Running unittests src/lib.rs (target/debug/deps/beta-[..])
>>> [CWD]/beta
<<< [CWD]/nightly
...
   Compiling nightly v0.1.0 ([CWD]/nightly)
    Finished `test` profile [unoptimized + debuginfo] target(s) in [..]s
     Running unittests src/lib.rs (target/debug/deps/nightly-[..])
>>> [CWD]/nightly
